index.js

```diff
@@ -46,43 +64,54 @@
 // console.log(sameSameButDifferent(5, 7)) 
 // // => "different"
 
-// console.log(sameSameButDifferent(123, "122")) 
+console.log(sameSameButDifferent(123, "122")) 
 // // => "different"
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 5 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)
+
+
+console.log("%cQuestion 5", "color: red")
+
+function updateGrade(student, grade) {
+
+}
```

Comment:
The key here was understanding that objects are reference types. Object values are references, so when we mutate the student object inside updateGrade, the change will persist outside the function.

```javascript
function updateGrade(studentObject, gradeNumber) {
  studentObject.grade = gradeNumber
}
```